static/js/utils/photometry.js

```diff
@@ -0,0 +1,89 @@
+/* global Plotly */
+
+const filter_mapper = {
+  ztfg: "green",
+  ztfr: "red",
+  ztfi: "orange",
+};
+
+function filter2color(filter) {
```

Comment:
That will avoid having every non-ztf blue, which is clearly a problem and blocker before we can use these public pages